src/components/modes/teacher/ImageUpload.js

```diff
@@ -12,42 +13,22 @@ import { MAX_FILE_SIZE } from '../../../config/settings';
 import { MUTATION_KEYS, useMutation, hooks } from '../../../config/queryClient';
 import { APP_SETTINGS } from '../../../constants/constants';
 
-const useStyles = makeStyles((theme) => ({
-  '@keyframes blinker': {
-    from: { opacity: 0 },
-    to: { opacity: 1 },
-  },
-  uploadingText: {
-    animation: '$blinker',
-    animationDuration: '2s',
-    animationTimingFunction: 'linear',
-    animationIterationCount: 'infinite',
-    fontSize: '0.9vw',
-  },
-  imageUploadContainer: {
-    display: 'flex',
-    alignItems: 'center',
-    justifyContent: 'space-between',
-    width: '100%',
-    marginTop: theme.spacing(2),
-    marginBottom: theme.spacing(1),
-  },
-  typographyContainer: {
-    display: 'flex',
-    flexDirection: 'column',
-  },
-  headerText: {
-    fontSize: '1.05vw',
-  },
-  captionText: {
-    fontSize: '0.8vw',
-    color: '#383838',
-  },
+const ImageUploadContainer = styled('div')(() => ({
+  display: 'flex',
+  alignItems: 'center',
+  justifyContent: 'space-between',
+  width: '100%',
+  marginTop: 2,
+  marginBottom: 1,
+}));
+
+const CloseButton = styled(Button)(() => ({
+  width: '20%',
+  fontSize: '1vw',
```

Comment:
I am not sure you want to set your fontSize depending on the `viewWidth` (`vw`)  of the viewport. See [this article](https://www.freecodecamp.org/news/css-unit-guide/) for some alternatives.